From 95c25299ec621bd120187cb6043891a2486c7d79 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Wed, 11 Nov 2020 10:42:55 +0100
Subject: [PATCH 2/2] config.mak.uname: remove unused NEEDS_SSL_WITH_CURL flag
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The NEEDS_SSL_WITH_CURL flag was still being set in one case, but
hasn't existed since 23c4bbe28e6 ("build: link with curl-defined
linker flags", 2018-11-03). Remove it, and a comment which referred to
it. See 6c109904bc8 ("Port to HP NonStop", 2012-09-19) for the initial
addition of the comment.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 config.mak.uname | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/config.mak.uname b/config.mak.uname
index 8d7b96f8b9b..5b30a9154ac 100644
--- a/config.mak.uname
+++ b/config.mak.uname
@@ -541,11 +541,6 @@ ifeq ($(uname_S),NONSTOP_KERNEL)
 	# removing the directory at OS releases J06.21 and L17.02.
 	# Default to the older rm until those two releases are deprecated.
 	RM = /bin/rm -f
-	# As detected by './configure'.
-	# Missdetected, hence commented out, see below.
-	#NO_CURL = YesPlease
-	# Added manually, see above.
-	NEEDS_SSL_WITH_CURL = YesPlease
 	NEEDS_CRYPTO_WITH_SSL = YesPlease
 	HAVE_DEV_TTY = YesPlease
 	HAVE_LIBCHARSET_H = YesPlease
-- 
2.29.2.222.g5d2a92d10f8

